<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OperationDialog</class>
 <widget class="QDialog" name="OperationDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>446</width>
    <height>511</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>446</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Operation</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="nameWidget" native="true">
     <layout class="QHBoxLayout" name="layerNameLayout">
      <item>
       <widget class="QLabel" name="nameLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>Na&amp;me</string>
        </property>
        <property name="buddy">
         <cstring>nameTextEdit</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="nameTextEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="resolutionWidget" native="true">
     <layout class="QHBoxLayout" name="resolutionLayout">
      <item>
       <widget class="QLabel" name="resolutionLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>&amp;Resolution</string>
        </property>
        <property name="buddy">
         <cstring>resolutionComboBox</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="resolutionComboBox">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>40</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>2048x2048</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>4096x4096</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="functionLayerWidget" native="true">
     <layout class="QHBoxLayout" name="functionLayerLayout">
      <property name="bottomMargin">
       <number>1</number>
      </property>
      <item>
       <widget class="QLabel" name="functionLayerLabel">
        <property name="text">
         <string>Function &amp;Layer</string>
        </property>
        <property name="buddy">
         <cstring>functionLayerComboBox</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="functionLayerComboBox"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="fieldWidget" native="true">
     <layout class="QHBoxLayout" name="fieldLayout">
      <property name="topMargin">
       <number>1</number>
      </property>
      <item>
       <spacer name="fieldSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>30</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="fieldLabel">
        <property name="text">
         <string>Fie&amp;ld</string>
        </property>
        <property name="buddy">
         <cstring>fieldComboBox</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="fieldComboBox"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="baseLayerWidget" native="true">
     <layout class="QHBoxLayout" name="baseLayerLayout">
      <property name="bottomMargin">
       <number>1</number>
      </property>
      <item>
       <widget class="QLabel" name="baseLayerLabel">
        <property name="text">
         <string>&amp;Base Layer</string>
        </property>
        <property name="buddy">
         <cstring>baseLayerComboBox</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="baseLayerComboBox"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="radiusWidget" native="true">
     <layout class="QHBoxLayout" name="radiusLayout">
      <property name="topMargin">
       <number>1</number>
      </property>
      <item>
       <widget class="QLabel" name="radiusLabel">
        <property name="text">
         <string>&amp;Radius</string>
        </property>
        <property name="buddy">
         <cstring>radiusSpinBox</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="radiusSpinBox">
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="commaLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>,</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="radiusMaxSpinBox">
        <property name="value">
         <double>2.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="operationWidget" native="true">
     <layout class="QHBoxLayout" name="operationLayout">
      <property name="bottomMargin">
       <number>1</number>
      </property>
      <item>
       <widget class="QLabel" name="operationLayerLabel">
        <property name="text">
         <string>Ope&amp;ration</string>
        </property>
        <property name="buddy">
         <cstring>operationLayerComboBox</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="operationLayerComboBox">
        <item>
         <property name="text">
          <string>Maximum</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Minimum</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Mean</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Variance</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Standard Deviation</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Null</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>No null</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="vectorWidget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <item>
       <widget class="QLabel" name="vecXLabel">
        <property name="text">
         <string>&amp;X</string>
        </property>
        <property name="buddy">
         <cstring>vecXSpinBox</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="vecXSpinBox">
        <property name="decimals">
         <number>16</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="vecYLabel">
        <property name="text">
         <string>Y</string>
        </property>
        <property name="buddy">
         <cstring>vecYSpinBox</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="vecYSpinBox">
        <property name="decimals">
         <number>16</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="vecZLabel">
        <property name="text">
         <string>&amp;Z</string>
        </property>
        <property name="buddy">
         <cstring>vecZSpinBox</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="vecZSpinBox">
        <property name="decimals">
         <number>16</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Minimum</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonExecute">
       <property name="text">
        <string>Create</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonExecute</sender>
   <signal>released()</signal>
   <receiver>OperationDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>348</x>
     <y>396</y>
    </hint>
    <hint type="destinationlabel">
     <x>266</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonCancel</sender>
   <signal>released()</signal>
   <receiver>OperationDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>438</x>
     <y>396</y>
    </hint>
    <hint type="destinationlabel">
     <x>266</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
